test/docs/cbd.test.ts

```diff
@@ -101,14 +101,11 @@ describe('Get Started (CBD PoC)', () => {
       },
     };
     const NFTBalance = new ContractCondition(NFTBalanceConfig);
-
-    const encrypter = newDeployed.encrypter;
-
+    const newConditions = new ConditionSet([NFTBalance]);
     const plaintext = 'this is a secret';
-    const encryptedMessageKit = encrypter.encryptMessagePre(
-      plaintext,
-      new ConditionSet([NFTBalance])
-    );
+    const encryptedMessageKit = newDeployed
+      .makeEncrypter(newConditions)
+      .encryptMessagePre(plaintext);
```

Comment:
```suggestion
    const newConditions = new ConditionSet([NFTBalance]);
    const plaintext = 'this is a secret';
    const encrypter = newDeployed.makeEncrypter(newConditions)
    const encryptedMessageKit = encrypter.encryptMessagePre(plaintext);
```

Just a reminder that this test is intended to reflect the code shown in documentation in order to track if it actually works.

https://docs.threshold.network/app-development/threshold-access-control-tac/get-started-with-tac#5.-encrypt-the-plaintext-and-update-conditions

So, a change here should involve a change in GitBook documentation.

That said, since this test is linked to the "Get Started" section, the code here shown should be as educational as possible. So I suggest to explicitly create the encrypter object.

Reply:
I see some merit in making this more explicit, I will update the code example.

We should definitely update GitBook docs to match these tests, this is probably something we want to do upon merging the epic branch (`tdec-epic`).